packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -44,7 +44,10 @@ export interface CommonAutoScalingGroupProps {
   /**
    * Initial amount of instances in the fleet
    *
-   * @default 1
+   * If this is set to a number, every deployment will reset the amount of
+   * instances to this number. It is recommended to leave this value blank.
+   *
+   * @default minCapacity, and leave unchanged during deployment
```

Comment:
Is this accurate, it doesn't look like you're defaulting desiredCapacity

Reply:
This describes the end-to-end behavior of what will happen if you leave the value empty.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-desiredcapacity 

>  If you do not specify a desired capacity, the default is the minimum size of the group.

Reply:
Add this link in the docstring